book/_config.yml

```diff
@@ -0,0 +1,47 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: Crafting beautiful maps with PyGMT
+author: The PyGMT Team
+logo: logo.svg
```

Comment:
Is it possible to make the logo clickable to follow the policy at https://www.egu.eu/visual-identity/?

Reply:
Went down a deep rabbit hole... Apparently it was possible in jupyter-book 0.7 but they did a refactor that removed this feature, see https://github.com/executablebooks/jupyter-book/issues/740.

Reply:
Still couldn't find a way to add a link to the logo after looking at https://jupyterbook.org/en/stable/advanced/sphinx.html, so I've just added a link to https://www.egu22.eu on the sidebar. Done in 896e55513644bd409656d7ca37f3572a16d61ef7 and looks like this now
![image](https://user-images.githubusercontent.com/23487320/167270962-9bc0b3ad-108e-4658-a270-1e4e86ac25ff.png)